### PR TITLE
fix: ensure matching shows when applicable, clean up initial visibili…

### PR DESCRIPTION
**Pull request by mcstover**
…ty logic

### CODE REVIEW

File: src/components/BorrowerProfile/LendCta.vue [hunk 1]
```diff
@@ -430,10 +431,10 @@ export default {
 			);
 		},
 		cycleStatsSlot() {
-			if (this.matchingText.length && !isMatchAtRisk) {
+			if (this.matchingText.length) {
 				const cycleSlotMachine = () => {
-					if (this.statScrollAnimation) {
-						this.statScrollAnimation = false;
+					if (!this.isMatchAtRisk) {
+						this.statScrollAnimation = !this.statScrollAnimation;
```
Comment (mcstover) [inline src/components/BorrowerProfile/LendCta.vue:437]: This covers the previous if/else condition and we only enable this swapping if the loan is matching and not at risk.

File: src/components/BorrowerProfile/LendCta.vue [hunk 2]
```diff
@@ -235,7 +235,7 @@
 							<span
 								class="tw-inline-block tw-align-middle"
 								key="loanMatchingText"
-								v-if="!statScrollAnimation"
+								v-if="!statScrollAnimation && !isMatchAtRisk"
```
Comment (mcstover) [inline src/components/BorrowerProfile/LendCta.vue:238]: Hide this section if matching is at risk

File: src/components/BorrowerProfile/LendCta.vue [hunk 3]
```diff
@@ -376,9 +376,6 @@ export default {
 
 			if (this.status === 'fundraising' && this.numLenders > 0) {
 				this.lenderCountVisibility = true;
-			}
-
-			if (this.lenderCountVisibility && this.matching !== '') {
```
Comment (mcstover) [inline src/components/BorrowerProfile/LendCta.vue:381]: This block wasn't needed b/c we can `statScrollAnimation` at the same time we determine `lenderCountVisibility` . Previously this check `(this.lenderCountVisibility && this.matching !== '')` was actually short-circuited b/c `this.matching` was always `undefined` thus we always activate `statScrollAnimation`.

File: src/components/BorrowerProfile/LendCta.vue [hunk 4]
```diff
@@ -430,10 +427,10 @@ export default {
 			);
 		},
 		cycleStatsSlot() {
-			if (this.matchingText.length && !isMatchAtRisk) {
```
Comment (mcstover) [inline src/components/BorrowerProfile/LendCta.vue:433]: This was broken b/c I forgot `this.` 🤦

**Review by emuvente — APPROVED** [review]

**Pull request merged**